### PR TITLE
chore: print warning logs about overwritten workflows

### LINKED ISSUE #111: Warn user that workflows are beeing overrided
When user reruns cycli on already configures project, let's warn him while overriding workflows. 
It would be good to add readme section for that: 
- what happens when you rerun on configured repo
- how to add just the new script

### PR DESCRIPTION
**Pull request by km1chno**
Resolves #111 

Adds warning log and information to readme (the WARNING box):

Zrzut ekranu 2024-09-19 o 13 45 16

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ then select the setup that matches your project - `npx setup-ci` generates workf
 
 If your project has a monorepo structure, run the script from the root directory of the app for which you want to set up workflows.
 
+> [!WARNING]
+> If you re-run `npx setup-ci` in the same project or you already have some workflows set up manually, they might get overwritten. Therefore we recommend running the script with no uncommited changes, so you can easily inspect the diff after script execution.
```
Comment (maciekstosio) [inline readme.md:22]:
```suggestion
> If you re-run `npx setup-ci` in the same project or already have some workflows set up manually, they might get overwritten. Therefore, we recommend running the script with no uncommitted changes, so you can easily inspect the diff after script execution.
```

File: src/extensions/workflows.ts [hunk 2]
```diff
@@ -45,6 +45,12 @@ module.exports = (toolbox: CycliToolbox) => {
       workflowFileName
     )
 
+    if (toolbox.filesystem.exists(target)) {
+      toolbox.interactive.warning(
+        `Workflow file ${workflowFileName} already exists and will be overwritten.`
```
Comment (maciekstosio) [inline src/extensions/workflows.ts:50]: I wonder if the workflow isn't something GitHub actions specific, although not sure if it's worth to worry about it now

Comment (km1chno) [reply src/extensions/workflows.ts:50]: gitlab calls them "pipelines"

Comment (km1chno) [reply src/extensions/workflows.ts:50]: but I agree we don't have to worry about it now

**Review by maciekstosio — APPROVED** [review]

**Pull request merged**